syncer/checkpoint.go

```diff
@@ -258,6 +266,8 @@ type RemoteCheckPoint struct {
 	globalPoint         *binlogPoint
 	globalPointSaveTime time.Time
 
+	safeModeExitPoint *binlog.Location
```

Comment:
How about adding some comments to explain?
- when it will be set and flush
- when it will be unset and flush
- when it will be used to judge

Reply:
https://github.com/pingcap/dm/pull/988/commits/d4705aa46cdda2d136f9812dd22a146d231fa2cc